src/modules/blockchain/implementation/web3-service.js

```diff
@@ -1294,6 +1294,7 @@ class Web3Service {
             this.scoringFunctionsContracts[2],
             'getParameters',
             [],
+            CONTRACTS.LINEAR_SUM,
```

Comment:
This won't work, we're not listening to events of this contract. Also getter `getParameters` returns multiple values, while setters are separate for each parameter, so cache should be cleared on change of any of the parameters, same goes for `Log2PLDSF` parameters caching